packages/okta-angular/README.md

```diff
@@ -1,19 +1,39 @@
+[Okta Auth SDK]: https://github.com/okta/okta-auth-js
+[@angular/router]: https://angular.io/guide/router
+[Observable]: https://angular.io/guide/observables
+[Dependency Injection]: https://angular.io/guide/dependency-injection
+[Auth service]: #oktaauthservice
+
 # Okta Angular SDK
 
 [![npm version](https://img.shields.io/npm/v/@okta/okta-angular.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-angular)
 [![build status](https://img.shields.io/travis/okta/okta-oidc-js/master.svg?style=flat-square)](https://travis-ci.org/okta/okta-oidc-js)
 
-An Angular wrapper around [Okta Auth JS](https://github.com/okta/okta-auth-js), that builds on top of Okta's [OpenID Connect API](https://developer.okta.com/docs/api/resources/oidc.html).
+Okta Angular SDK builds on top of the [Okta Auth SDK][]. This SDK adds integration with [@angular/router][] and provides additional logic and components designed to help you quickly add authentication and authorization to your Angular single-page web application.
+
+With the [Okta Auth SDK][], you can:
+
+- Login and logout from Okta using the [OAuth 2.0 API](https://developer.okta.com/docs/api/resources/oidc)
+- Retrieve user information
+- Determine authentication status
+- Validate the current user's session
+
+All of these features are supported by this SDK. Additionally, using this SDK, you can:
+
+- Add "protected" routes, which will require authentication before render
+- Define custom logic/behavior when authentication is required
+- Subscribe to changes in authentication state using an [Observable] property
+- Provide an instance of the [Auth service][] to your components using [Dependency Injection][]
+
+> This SDK does not provide any UI components.
 
 This library currently supports:
 
 - [OAuth 2.0 Implicit Flow](https://tools.ietf.org/html/rfc6749#section-1.3.2)
 - [OAuth 2.0 Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1)
 - [Proof Key for Code Exchange (PKCE)](https://tools.ietf.org/html/rfc7636)
 
-This library has been tested for compatibility with the following Angular versions: 4, 5, 6, 7, 8
-
-`okta-angular` works directly with [`@angular/router`](https://angular.io/guide/router).
+> This library has been tested for compatibility with the following Angular versions: 4, 5, 6, 7, 8
```

Comment:
Where do we stand on Angular 9?

Reply:
v1.2.2 of our Angular SDK works with Angular 9. v1.3.0 does not. See https://github.com/okta/okta-oidc-js/issues/643 for more info.